discord/guild.py

```diff
@@ -1906,17 +1913,26 @@ async def edit(
 
             fields['system_channel_flags'] = system_channel_flags.value
 
+        features = []
```

Comment:
might be worth at this point defaulting features to all the features the guild has, and adding/removing as appropriate.

Reply:
Thinking more on this, I'm pretty sure that's necessary and the only way to safely do this.

Since if a new editable feature not supported by dpy is added it'll be blown away every time someone updates other features if we keep this as is.

Reply:
Using prior state is always going to be buggy. For features in particular though it doesn't allow most features to be in the patch request so I don't think your suggestion is the way to go.

Reply:
It seems the client just sends the entire feature list now? Unless I missed something.

Reply:
If they finally changed it so it allows you to pass in the entire list that's good though it also feels finicky. I'll have to ask for clarification later.

Reply:
I asked and passing the entire list is the way to go now, so might as well make that change.

Reply:
Changed to pass the entire list.